src/util/caseConv.ts

```diff
@@ -80,20 +78,21 @@ function toSnakeCase<T>(obj: any, options: ConvertOptions = {}): T {
         if (options.cast?.[k]) {
             try {
                 const cls = options.cast[k];
-                // @ts-ignore
```

Comment:
Have you checked that this compiles?

Reply:
I didn't write this, but yes it correctly compiles.